### DISCUSSION TITLE
Editor Replace Button Hotkey

### DISCUSSION THREAD

**Discussion by rohitab** [FarGroup/FarManager#616] [General]
Most dialogs in _Far_ do not have hotkeys for the default button, since you can just press `Enter`. For example, in the _Search_ dialog in Viewer/Editor/Help, the _Search_ button does not have a hotkey. However, in the _Replace_ dialog in the Editor, the _Replace_ button has an `Alt+R` hotkey.

What do you think about removing the hotkey from the _Replace_ button and assigning `Alt+R` to the _Replace with_ edit control instead? It seems natural to me, to use `Alt+S` and `Alt+R` to switch between _Search_ and _Replace_ with boxes. It might inconvenience people that are accustomed to using the `Alt+R`/`Alt+E` hotkeys though.

> [MKadaner](https://github.com/MKadaner): I also noticed this discrepancy. I think this is a good idea. The decision is mostly for the language maintainers, except for there is no dedicated maintainer for English localization.

**Comment by MKadaner** [comment]
Thank you for reporting, @rohitab. @alabuzhev if you are fine with this breaking change, please assign the issue to me. Otherwise, consider closing it with prejudice. 🙂

**Comment by alabuzhev** [comment]
@MKadaner #618
Please [write a comment](https://github.blog/changelog/2019-06-25-assign-issues-to-issue-commenters/) there to make it assignable to you.

**Comment by w17** [comment]
All things in .lng file.
So you can adjust it personally in FarEng.lng.custom

**Comment by w17** [comment]
Look Changelog comment for build 4522.

**Comment by MKadaner** [comment]
Nice feature, @w17. I wonder though how many users are aware of it. ;)

**Reply by MKadaner** [reply]
Should it be described in help, by the way?

**Reply by w17** [reply]
it would be great.

**Reply by MKadaner** [reply]
Done in #628.

**Comment by rohitab** [comment]
Excellent feature. I had no idea. I've been using Lua to do these customizations. This is so much easier.

Would definitely be useful to have this information in _Help_.

**Reply by MKadaner** [reply]
OK, I can do that. Any suggestions where to put it though. Interestingly, there is no likely section; neither _Languages_, nor _Localization_, nor _Customization_, nor even generic _Useful features and everything else_. I am thinking about a new article _Localization: Customizing Dialogs_.
